.github/workflows/build-sub-projects.yml

```diff
@@ -17,6 +17,7 @@ jobs:
       - run: |-
           cd frontend/
           yarn install --check-files --frozen-lockfile
+          yarn run lint
```

Comment:
please add a step to fail the build if finding the mutations,

https://github.com/awslabs/clickstream-analytics-on-aws/blob/73a1e174a4e4cbcc0b29c7773d0751540a539fc9/.github/workflows/build.yml#L74-L78

Reply:
```
    - git diff --staged --patch --exit-code > .repo.patch || export mutations_happened=true
    - if [ "$mutations_happened" = "true" ]; then cat .repo.patch && exit 1; fi;
 ```